UPGRADE-3.1.md

```diff
@@ -103,11 +103,10 @@ FrameworkBundle
 
    ```yaml
    framework:
-       serializer: ~
```

Comment:
I agree with @nicolas-grekas here. In this particular example I think it's better to show it like this.